### ISSUE TITLE
Can't create LetsEncrypt certificate unless nginx is installed,running & configured for domain

### ISSUE THREAD

**Issue by picklit** [serghey-rodin/vesta#1174]
### Operating System (OS/VERSION):

CentOS 7

### VestaCP Version:

Type here, e.g. 3.14159

### Installed Software (what you got with the installer):

apache, mysql

### Steps to Reproduce:

Create a new domain and attempt to create a 

### Other Notes:

Error returned is: Error: Invalid response from http://test.pickl-it.com/.well-known/acme-challenge/8pF0lUC2qAEkKz7VlEGKYduBpXJ1rlZfFQluUrkHEWs: \

Anything else?

**Comment by madeITBelgium** [comment]
Can you check if the DNS and the email address of the user are correct?

**Comment by picklit** [comment]
Yes, both are correct.   The domain has a CNAME rather than A record, but it doesn't seem like that should matter.  I did resolve this on another server by configuring NGINX as the proxy for the domain.  For this particular installation I'm testing I chose not to install NGINX, now there doesn't even seem to be a work-around.   Also, the .well-known directory never seems to be created, so understandably it can't be read remotely.
